packages/base/src/3dview/mainview.tsx

```diff
@@ -123,7 +125,9 @@ export class MainView extends React.Component<IProps, IStates> {
       firstLoad: true,
       wireframe: false,
       transform: false,
-      clipEnabled: false,
+      clipEnabled: true,
```

Comment:
Probably a rebase issue?

```suggestion
      clipEnabled: false,
```